CHANGELOG.yaml

```diff
@@ -1,3 +1,9 @@
+master:
+  fixed bugs:
```

Comment:
Then why's this PR fixing a bug where `Cookie. toString` is not serializing the `extensions` properly?

Reply:
Removed the `extensions` related part, will raise as part of separate PR.